addon/helpers/-element.js

```diff
@@ -21,7 +22,20 @@ export default Helper.extend({
         // ignore
       }
 
-      throw new Error(message);
+      try {
+        throw new Error(message);
+      } catch (e) {
+        if (typeof Ember.onerror === 'function') {
+          Ember.onerror(e);
+        } else {
+          console.assert(false, e); // eslint-disable-line no-console
+        }
+      }
```

Comment:
The issue with follow on errors after something throws during render has been fixed in https://github.com/glimmerjs/glimmer-vm/pull/1073 (and ported to 3.16, 3.18, and 3.19). Also you can work around it in tests by making sure to wrap the thing that throws an error with a stable element.